### PR TITLE
Fix few typos in comments

### PR DESCRIPTION
**Pull request by jbrianceau**
Note that this change only impacts comments and metadata, not actual code.

### CODE REVIEW

File: test/catch.hpp [hunk 1]
```diff
@@ -10545,7 +10545,7 @@ namespace Catch {
             // Extracts the actual name part of an enum instance
             // In other words, it returns the Blue part of Bikeshed::Colour::Blue
             StringRef extractInstanceName(StringRef enumInstance) {
-                // Find last occurence of ":"
+                // Find last occurrence of ":"
```
Comment (kennykerr) [inline test/catch.hpp:10548]: This file is generated by https://github.com/catchorg/Catch2 - please make the contribution there and then I'll be happy to ingest a new version from that project.

**Review by kennykerr — CHANGES_REQUESTED** [review]
Thanks for the fixes! Happy to accept them, but please revert the change to catch.hpp.

**Comment by jbrianceau** [comment]
> Thanks for the fixes! Happy to accept them, but please revert the change to catch.hpp.

Sure, I've just updated the PR accordingly.

File: test/catch.hpp (hunk 1 above)
Comment (jbrianceau) [reply test/catch.hpp]: Pull request is out: https://github.com/catchorg/Catch2/pull/2206
I don't think that it's worth ingesting a new version just for such a small cosmetic change and I definitely won't be offended if you wait for a more valuable payload before making a new ingestion :)

**Review by kennykerr — APPROVED** [review]
Thanks!

**Pull request merged**